starter-code/apple-pie-starter-code/css/main.css

```diff
@@ -0,0 +1,49 @@
+header{
+    border-style: solid;
+}
+body {
+    border-style: solid;
+    margin: 200px;
+}
+.presentation1 {
+    border-style: solid;
+    position: relative;
+}
+.h1-centered{
+    border-style: solid;
+    position: absolute;
+    top: 30%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    color: white;
+    font-size: 200%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+
+.presentation2 {
+    border-style: solid;
+    padding: 10%;
+    text-align: justify;
+    padding: -50xp;
+    font-size: 120%;
+    font-family: Arial, Helvetica, sans-serif;
+}
+.image1{
+    border-style: solid;
+    max-width: 100%; overflow: hidden;
+    text-indent: -100px -300px -100px -100px;
```

Comment:
try not to use negtive value as it will be a mess if you are building a responsive website, instead try to use the position & the flexbox method that we've seen yesterday